docs/emr_deployment.md

```diff
@@ -0,0 +1,126 @@
+---
+id: emr_deployment
+title: GeoTrellis EMR deployment with GDAL
+sidebar_label: GeoTrellis EMR deployment with GDAL
+---
+
+GeoTrellis can be used with _EMR 6_. However, if there is a need in GDAL bindings usage there can be some certain complexities.
+This section describes the entire _EMR 6_ deployment and GDAL installation in a way that would satisfy GeoTrellis GDAL requirements.
+
+Starting _GeoTrellis_ `3.4.2`, _GeoTrellis_ is compatible only with the GDAL`3.1 +` versions.
+
+## Installing GDAL on EMR
+
+The EMR installation conatans of the following steps:
+
+1. Install _GDAL_
+2. Set Spark `LD_LIBRARY_PATH`. In this case `GeoTrellis GDAL` would be able to use the installed `GDAL`.
+
+You can also check out [GeoTrellis Spark template](https://github.com/geotrellis/geotrellis-spark-job.g8) that is properly configured for the EMR jobs.
+
+### EMR Bootsrap script (GDAL installation on EMR)
+
+The EMR bootstrap script with all appropriate settings is located under the following S3 URI: [s3://geotrellis-test/emr-gdal/bootstrap.sh](s3://geotrellis-test/emr-gdal/bootstrap.sh).
+It installs GDAL 3.1.2 (by default) through conda and sets all appropriate env variables in case 
+it would be required to log in onto nodes and experiment with the `spark-shell`.
+
+```bash
+#!/bin/bash
+
+set -ex
+
+# The default GDAL version would be 3.1.2
+GDAL_VERSION=$1
+GDAL_VERSION=${GDAL_VERSION:="3.1.2"}
+
+# Install Conda
+wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh
+sudo sh Miniconda-latest-Linux-x86_64.sh -b -p /usr/local/miniconda
+
+source ~/.bashrc
+export PATH=/usr/local/miniconda/bin:$PATH
+
+# Install GDAL
+conda config --add channels conda-forge
+sudo pip3 install tqdm && \
+sudo /usr/local/miniconda/bin/conda install python=3.6 -y && \
+sudo /usr/local/miniconda/bin/conda install -c anaconda hdf5 -y && \
+sudo /usr/local/miniconda/bin/conda install -c conda-forge libnetcdf gdal=${GDAL_VERSION} -y
+
+echo "export PATH=/usr/local/miniconda/bin:$PATH" >> ~/.bashrc
+echo "export LD_LIBRARY_PATH=/usr/local/miniconda/lib/:/usr/local/lib:/usr/lib/hadoop/lib/native:/usr/lib/hadoop-lzo/lib/native:/docker/usr/lib/hadoop/lib/native:/docker/usr/lib/hadoop-lzo/lib/native:/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib" >> ~/.bashrc
+```
+
+#### Spark settings (optional)
+
+Set `LD_LIBRARY_PATH` for spark, through the `spark-defaults` settings:
+
+```json
+{
+  "Classification": "spark-defaults",
+  "Properties": {
+    "spark.yarn.appMasterEnv.LD_LIBRARY_PATH": "/usr/local/miniconda/lib/:/usr/local/lib",
+    "spark.executorEnv.LD_LIBRARY_PATH": "/usr/local/miniconda/lib/:/usr/local/lib"
+  }
+}
+```
+
+These parameters (above) can be also set / overrided via the `spark-submit --conf` settings.
+
+### SBT Lighter plugin configuration example
```

Comment:
Before showing the complete example, it would be nice to provide a list of SBT lighter settings that _must_ be set for the bootstrap script to work. Presumably that's the `BootstrapAction` and the two spark-defaults that set `LD_LIBRARY_PATH`, but it's not totally clear from the example. 

Perhaps then a better title would be something like `### Configuration for SBT Lighter Plugin`